### PR TITLE
Create "Sigstore’s cosign and policy-controller with GKE, Artifact Registry and KMS" blog article

### PR DESCRIPTION
**Pull request by mathieu-benoit**
I recently learned about using both [Sigstore's `cosign` and `policy-controller` with GKE/GAR/KMS](https://medium.com/google-cloud/sigstores-cosign-and-policy-controller-with-gke-and-kms-7bd5b12672ea), and found out that it was easier than what I would have expected. So I would like to propose to share this story with the Sigstore community via the Sigstore blog.

Let me know what you think, if you have any feedback, and if it fits with the purpose of this blog to have such stories. Thank you for your review.

### CODE REVIEW

**Comment by dlorenc** [comment]
This looks great to me, but I'm not an approver!

**Review by lukehinds — APPROVED** [review]
Thanks @mathieu-benoit , lgtm. Will get other TSC members to review / approve.

@bobcallaway / @trevrosen @priyawadhwa @SantiagoTorres ^^

**Review by bobcallaway — APPROVED** [review]

File: content/cosign-and-policy-controller-with-gke-artifact-registry-and-cloud-kms.md
Comment (trevrosen) [inline content/cosign-and-policy-controller-with-gke-artifact-registry-and-cloud-kms.md:13]: I would hesitate for us to call ourselves a "standard", as we're definitely not one (yet). 😄

**Review by trevrosen — CHANGES_REQUESTED** [review]
This is a great post - thank you! I have one request which is to replace the use of the word "standard" with something like "method" or "toolkit", as the word "standard" has a specific meaning in OSS and Sigstore does not yet meet that definition.

File: content/cosign-and-policy-controller-with-gke-artifact-registry-and-cloud-kms.md
Comment (dlorenc) [reply content/cosign-and-policy-controller-with-gke-artifact-registry-and-cloud-kms.md:13]: I'm guessing this is coming straight from the main website at https://www.sigstore.dev/ :
image

Comment (mathieu-benoit) [reply content/cosign-and-policy-controller-with-gke-artifact-registry-and-cloud-kms.md:13]: Yep, exactly, @trevrosen do you still want me to replace `standard` in the context of this blog post?

Comment (trevrosen) [reply content/cosign-and-policy-controller-with-gke-artifact-registry-and-cloud-kms.md:13]: Fair point - we should probably have a look at that wording too! 😄

Comment (trevrosen) [reply content/cosign-and-policy-controller-with-gke-artifact-registry-and-cloud-kms.md:13]: I'd forgotten that that's on the front page. 😅 I'll work through my questions around that elsewhere.

Comment (mathieu-benoit) [reply content/cosign-and-policy-controller-with-gke-artifact-registry-and-cloud-kms.md:13]: Out of curiosity, anything missing on my end to get your approval on this PR @trevrosen?

**Comment by lukehinds** [comment]
@priyawadhwa @SantiagoTorres @bobcallaway help us hit the quota of three so we can ship.

**Review by bobcallaway — APPROVED** [review]

**Comment by mathieu-benoit** [comment]
One more needed :), who wants to take it @priyawadhwa or @SantiagoTorres? Thanks!

**Review by priyawadhwa — APPROVED** [review]

**Review by SantiagoTorres — APPROVED** [review]

**Comment by mathieu-benoit** [comment]
Thanks everyone for your review! I think we are good to go here, isn't it?

**Pull request merged**
